src/library/objects/Messengers.js

```diff
@@ -24,7 +24,7 @@ Sample Usage:
 (function(){
 	"use strict";
 	
-	console.info("KC3改 Messengers loaded");
+	console.log("KC3改 Messengers loaded");

```

Comment:
oh sorry the real motivation was that it should also be removed by the build for webstore, where only .logs are removed. I wasn't aware of the collect errors. I'm fine with .info either way. I can just configure the build to also remove .info since if that collection only works for unpacked ones, no need to keep it on webstore.